shipitscript/shipitscript/script.py

```diff
@@ -61,15 +62,15 @@ def create_new_release_action(context):
         # TODO quit early, mark task as green though
         pass
     log.info('Determining most recent shippable revision')
-    shippable_revision = ship_actions.get_shippable_revision(
-        repo, last_shipped_revision
+    should_build = ship_actions.should_ship_revision(
+        repo, last_shipped_revision, current_revision,
     )
-    if not shippable_revision:
+    if not should_build:
         # TODO quit early, mark task as green though
         pass
```

Comment:
we need to do something here.

Reply:
rebased against my changes which already take care of this.